Neos.Fusion.Form/Classes/FusionObjects/FieldComponentImplementation.php

```diff
@@ -0,0 +1,166 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Fusion\Form\FusionObjects;
+
+/*
+ * This file is part of the Neos.Fusion.Form package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Fusion\Exception;
+use Neos\Fusion\FusionObjects\ComponentImplementation;
+use Neos\Fusion\Form\Domain\Model\FormDefinition;
+use Neos\Fusion\Form\Domain\Model\FieldDefinition;
+use Neos\Utility\ObjectAccess;
+
+/**
+ * Class FormFieldImplementation
+ * @package Neos\Fusion\Form\FusionObjects
+ */
+class FieldComponentImplementation extends ComponentImplementation
```

Comment:
Does it make sense to *extend* ComponentImplementation? Can we somehow compose the two objects in Fusion? (Or does that not make sense because this makes ergonomics in Fusion more difficult?)

I don't fully grasp the consequences of that yet :)

Reply:
Problem is that we need Form and Field  in context before content is evaluated. Which on the other hand is passed to the renderer as props.content.

Kind of a chicken and egg problem.

Reply:
@sebastian i managed to replace this with something like 
```
form = Neos.Fusion:FormDefinition
@context.form = ${this.form}
```
This makes sure that form is evaluated first and put into the context `form` that is available within content and renderer across component boundaries.

Reply:
nice ❤️